doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -30,6 +30,15 @@ Changelog
 
 The following sections provide detailed lists of changes by component.
 
+MCUboot
+=======
+
+* Added:
+
+  * An option to prevent inclusion of default nRF5340 network core DFU image hook, which allows a custom implementation by users if the :kconfig:option:`CONFIG_BOOT_IMAGE_ACCESS_HOOK_NRF5340` Kconfig option is disabled (enabled by default).
+    CMake can be used to add additional hook files.
+  * Support for multi-image DFU without using external flash on nRF53.
```

Comment:
```suggestion
  * An option to prevent the inclusion of the default nRF5340 network core DFU image hook, which allows a custom implementation by users if the :kconfig:option:`CONFIG_BOOT_IMAGE_ACCESS_HOOK_NRF5340` Kconfig option is disabled (it is enabled by default).
    CMake can be used to add additional hook files.
  * Support for multi-image DFU without using external flash memory on nRF53.
```